### ISSUE TITLE
Adding new app with github url on developer admincp console issue

### ISSUE THREAD

**Issue by Ylmzef** [moxi9/phpfox#716]
I'm trying adding some moxi9 apps with github url on developer admincp console but i'm getting "Server does not support git." error message.

Git are installed successfully on my server.

> root@s [~]# git --version
> git version 2.5.0-rc2

Centos 7
Tested git repos: 
https://github.com/moxi9/phpfox-shoutbox
https://github.com/moxi9/phpfox-base

**Comment by AnthonyKing** [comment]
have you tried adding an app without using the github url

**Comment by Ylmzef** [comment]
@AnthonyKing Why you do unnecessary comments on all of my tickets?

**Comment by Jinkers** [comment]
They wont install as the apps install via a specific routine that is created when exported by the developer - regardless if you use straight git commands. The only other way is if there is a app.lock file with the repo which is not the case.

**Comment by Data66PF** [comment]
Marked a bug to either remove the option or make it work with github only for test apps. However, making it work this way opens other issues. Recommend removing the option.

**Comment by robert9592** [comment]
Fixed for v4.1.1

**Comment by Ylmzef** [comment]
I'm getting now this error message: "Not a valid Git app." with this Git url: https://github.com/moxi9/phpfox-sms-captcha.git

```
## git version: ##
git version 2.5.0-rc2
## App JSON File: ##
stdClass Object
(
    [id] => PHPfox_RingCaptcha
    [name] => SMS Captcha
    [icon] => //d2h79mkp7etn4r.cloudfront.net/icon/2015/07/ae5c525be08bb5022cda99cc1d82af47.png
    [version] => 1.0
    [requires] => stdClass Object
        (
            [phpfox] => >= 4.0.2
        )

    [routes] => stdClass Object
        (
            [/ringcaptcha-verify] => stdClass Object
                (
                    [url] => {{ _url }}extend/ringcaptcha?page=verify
                )

            [/ringcaptcha-popup] => stdClass Object
                (
                    [url] => {{ _url }}extend/ringcaptcha?page=popup
                )

        )

    [settings] => stdClass Object
        (
            [ringcaptcha_enabled] => stdClass Object
                (
                    [info] => RingCaptcha Enabled
                    [type] => input:radio
                    [value] => 0
                    [js_variable] => 1
                )

            [ringcaptcha_app_key] => stdClass Object
                (
                    [info] => App Key
                )

            [ringcaptcha_secret_key] => stdClass Object
                (
                    [info] => Secret Key
                )

            [ringcaptcha_name] => stdClass Object
                (
                    [info] => Sender Name (Max 11 alpha characters)
                    [value] => PHPfox
                )

        )

    [_url] => http://store.phpfox.us/
)
## Github Headers: ##
Array
(
    [0] => HTTP/1.1 200 OK
    [1] => Content-Security-Policy: default-src 'none'
    [2] => X-XSS-Protection: 1; mode=block
    [3] => X-Frame-Options: deny
    [4] => X-Content-Type-Options: nosniff
    [5] => Strict-Transport-Security: max-age=31536000
    [6] => ETag: "ff3dde88319bcb843dc1e7e0780824e56f40cf46"
    [7] => Content-Type: text/plain; charset=utf-8
    [8] => Cache-Control: max-age=300
    [9] => X-GitHub-Request-Id: 17EB2B22:4487:15025A3:5667B0BF
    [10] => Content-Length: 792
    [11] => Accept-Ranges: bytes
    [12] => Date: Wed, 09 Dec 2015 04:40:32 GMT
    [13] => Via: 1.1 varnish
    [14] => Connection: close
    [15] => X-Served-By: cache-ams4131-AMS
    [16] => X-Cache: MISS
    [17] => X-Cache-Hits: 0
    [18] => Vary: Authorization,Accept-Encoding
    [19] => Access-Control-Allow-Origin: *
    [20] => X-Fastly-Request-ID: 051147b3dba00f879401f780ff2940d5a4ebc7c3
    [21] => Expires: Wed, 09 Dec 2015 04:45:32 GMT
    [22] => Source-Age: 0
)




```
